WalletWasabi.Fluent/Models/FeeDisplayFormat.cs

```diff
@@ -6,6 +7,6 @@ public enum FeeDisplayFormat
 {
 	BTC,
 
-	[FriendlyName("sat/vByte")]
-	SatoshiPerByte,
-}
+	[FriendlyName("sats")]
+	Satoshis,
```

Comment:
IMO it should be `Satoshi`.

Reply:
+1, as a user, I'd prefer `Satoshi` as well